axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/bankreconciliation/BankReconciliationServiceImpl.java

```diff
@@ -1564,4 +1588,15 @@ public void mergeSplitedReconciliationLines(BankReconciliation bankReconciliatio
       }
     }
   }
+
+  protected BigDecimal getConvertedAmount(BigDecimal value, BankReconciliation bankReconciliation)
+      throws AxelorException {
+    BigDecimal currencyRate =
+        currencyService.getCurrencyConversionRate(
+            bankReconciliation.getCurrency(),
+            bankReconciliation.getCompany().getCurrency(),
+            dateService.date());
+
+    return value.divide(currencyRate, RoundingMode.HALF_UP);
```

Comment:
same issue as above, we should call `getAmountCurrencyConvertedAtDate`. Here we are missing the scale